ikabot/function/distributeResources.py

```diff
@@ -158,99 +171,104 @@ def distribute_evenly(s, resource):
+			city = getCity(html)
+			if resource_type == 1:
+				city['disponible'] = city['recursos'][resource_type] - city['consumo'] - 1
 			else:
-				ciudad['disponible'] = ciudad['recursos'][recurso]
-			if ciudad['disponible'] < 0:
-				ciudad['disponible'] = 0
-			recursoTotal += ciudad['disponible']
-			ciudadesOrigen[idCiudad] = ciudad
+				city['disponible'] = city['recursos'][resource_type]
+			if city['disponible'] < 0:
+				city['disponible'] = 0
+			resources_total += city['disponible']
+			origin_cities[city_id] = city
 		else:
-			html = s.get(urlCiudad + idCiudad)
-			ciudad = getCity(html)
-			ciudad['disponible'] = ciudad['freeSpaceForResources'][recurso]
-			if ciudad['disponible'] > 0:
-				ciudadesDestino[idCiudad] = ciudad
+			html = session.get(city_url + city_id)
+			city = getCity(html)
+			city['disponible'] = city['freeSpaceForResources'][resource_type]
+			if city['disponible'] > 0:
+				destination_cities[city_id] = city
 
-	if recursoTotal <= 0:
+	if resources_total <= 0:
 		print(_('\nThere are no resources to send.'))
 		enter()
 		return None
-	if len(ciudadesDestino) == 0:
+	if len(destination_cities) == 0:
 		print(_('\nThere is no space available to send resources.'))
 		enter()
 		return None
 
-	recursoXciudad = recursoTotal // len(ciudadesDestino)
-	espacios_disponibles = [ ciudadesDestino[city]['disponible'] for city in ciudadesDestino ]
-	totalEspacioDisponible = sum( espacios_disponibles )
-	restanteAEnviar = min(recursoTotal, totalEspacioDisponible)
+	resources_per_city = resources_total // len(destination_cities)
+	spaces_available = [ destination_cities[city]['disponible'] for city in destination_cities ]
+	total_spaces_available = sum( spaces_available )
+	remaining_resources = min(resources_total, total_spaces_available)
 	toSend = {}
 
-	while restanteAEnviar > 0:
+	while remaining_resources > 0:
 		len_prev = len(toSend)
-		for city in ciudadesDestino:
-			ciudad = ciudadesDestino[city]
-			if city not in toSend and ciudad['disponible'] < recursoXciudad:
-				toSend[city] = ciudad['disponible']
-				restanteAEnviar -= ciudad['disponible']
+		for city in destination_cities:
+			city = destination_cities[city]
+			if city not in toSend and city['disponible'] < resources_per_city:
+				toSend[city] = city['disponible']
+				remaining_resources -= city['disponible']
 
 		if len(toSend) == len_prev:
-			for city in ciudadesDestino:
+			for city in destination_cities:
 				if city not in toSend:
-					toSend[city] = recursoXciudad
+					toSend[city] = resources_per_city
 			break
 
-		espacios_disponibles = [ ciudadesDestino[city]['disponible'] for city in ciudadesDestino if city not in toSend ]
-		totalEspacioDisponible = sum( espacios_disponibles )
-		restanteAEnviar = min(restanteAEnviar, totalEspacioDisponible)
-		recursoXciudad = restanteAEnviar // len(espacios_disponibles)
+		spaces_available = [ destination_cities[city]['disponible'] for city in destination_cities if city not in toSend ]
```

Comment:
also `spaces` here, should be free capacity probably

Reply:
agree